src/lib/component/partial/Label/Label.js

```diff
@@ -31,33 +49,89 @@ const styles = theme => ({
   },
 });
 
+const imageExtensions = [
+  ".PNG",
+  ".JPG",
+  ".JPEG",
+  ".GIF",
+  ".WEBP",
+  ".TIFF",
+  ".PSD",
+  ".RAW",
+  ".BMP",
+  ".HEIF",
+  ".INDD",
+  ".SVG.",
+];
+
 class Label extends React.PureComponent {
   static propTypes = {
     classes: PropTypes.object.isRequired,
     name: PropTypes.string.isRequired,
     value: PropTypes.string.isRequired,
   };
 
-  parseLink = value => {
+  parseLink = (value, name) => {
     try {
       new URL(value);
+      // see if it's got an image extension
+      let results = imageExtensions.filter(ext =>
+        value.toUpperCase().includes(ext),
```

Comment:
I think it would be preferable to use [`endsWith`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith), that way we don't unintentionally match a URL like `/~/files/totally-an-image.png._________________________.exe`.

Reply:
On a related note, since we are already parsing the value on L76, we could use `pathname` property on the `URL`. That was we avoid matching what may look like a file extension in the query string or anchor.

eg.

```
const url = new URL(value);
const result = imageExtensions.some(ext => url.pathname.toUpperCase().endsWith(ext));
if (result) console.debug("tada! 🎉");
```

Reply:
@dabria I think my comment here still applies. It is pretty common for someone to add a query on the end of an image URL for cache busting purposes, (eg. https://my-cdn.com/some-img.png?somerandval).

<img width="447" alt="Screen Shot 2020-03-13 at 1 36 30 PM" src="https://user-images.githubusercontent.com/194892/76657710-b99a8000-652f-11ea-82a3-876c2ac43490.png">